client/src/components/AvatarDisplay/AvatarDisplay.tsx

```diff
@@ -1,15 +1,29 @@
+import { Badge } from '@material-ui/core';
 import Avatar from '@material-ui/core/Avatar';
 import { Profile } from '../../interface/Profile';
-import { User } from '../../interface/User';
+import useStyles from './useStyles';
 
 interface Props {
   loggedIn?: boolean;
-  user?: User | null | undefined;
+  online?: boolean;
+  offline?: boolean;
   profile?: Profile | null | undefined;
   src?: string;
 }
 
-const AvatarDisplay = ({ profile, src }: Props): JSX.Element => {
+const AvatarDisplay = ({ profile, src, online, offline }: Props): JSX.Element => {
+  const classes = useStyles();
+  if (online || offline)
+    return (
+      <Badge
+        classes={{ badge: `${classes.badge}  ${online && classes.offline} ${online && classes.online}` }}
```

Comment:
`{online && classes.offline}`, I think what you mean here is `{offline && classes.offline}` ?